### DISCUSSION TITLE
macOS port

### DISCUSSION THREAD

**Discussion by AlexeyKulagin** [evilsocket/opensnitch#979] [Q&A] [answered]
### Hello! This is the cool project!

**There's a question:** 
Are you planning to create a port for **macOS** users to install Opensnitch through **brew** for ex? It would be a really nice!

Thanks! And All the best!

**Comment by gustavo-iniguez-goya** [comment, marked as answer]
hey @manintech ,

No, that is not planned. Lulu and Little Snitch are already available for macOS, and they work really reall well.
